lib/ffmpeg/movie.rb

```diff
@@ -13,13 +13,15 @@ class Movie
 
     UNSUPPORTED_CODEC_PATTERN = /^Unsupported codec with id (\d+) for input stream (\d+)$/
 
-    def initialize(path)
+    def initialize(path, skip_head_request = true)
       @path = path
 
       if remote?
-        @head = head
-        unless @head.is_a?(Net::HTTPSuccess)
-          raise Errno::ENOENT, "the URL '#{path}' does not exist or is not available (response code: #{@head.code})"
+        if !skip_head_request
```

Comment:
```suggestion
       unless skip_head_request
```

Reply:
I prefer to don't use `unless` at all. Makes it harder to understand the code.